### PR TITLE
Remove link to nonexistent fb page

### PR DESCRIPTION
**Pull request by scottrigby**
Simple fix. We don't have a fb page
fb sharing still works fine

### CODE REVIEW

**Review by RedbackThomson — COMMENTED** [review]
Planning on adding Facebook back in the future? Otherwise probably delete the corresponding SVG - https://github.com/scottrigby/ogo-website/blob/remove-fb/src/svg/facebook.svg

**Comment by scottrigby** [comment]
> Planning on adding Facebook back in the future? Otherwise probably delete the corresponding SVG - https://github.com/scottrigby/ogo-website/blob/remove-fb/src/svg/facebook.svg

Removed w co-authorship credit. Thx!

**Review by roberthstrand — APPROVED** [review]
LGTM

**Review by RedbackThomson — APPROVED** [review]

File: gatsby-config.js [hunk 1]
```diff
@@ -11,7 +11,6 @@ module.exports = {
       slack: "https://slack.cncf.io",
       twitter: "https://twitter.com/opengitops",
       reddit: "https://www.reddit.com/r/GitOps",
-      facebook: "https://facebook.com/open-gitops",
```
Comment (lloydchang) [inline gatsby-config.js:14]: @scottrigby Facebook says:
> "the page may have been removed."

Any jokes? 😆
> "I had a GitOps joke, which I accidentally deleted it. But I think it'll be back in a minute."
> by @chanwit https://twitter.com/chanwit/status/1287110405907550208

**Review by lloydchang — APPROVED** [review]

**Review by chrispat — APPROVED** [review]

File: gatsby-config.js (hunk 1 above)
Comment (scottrigby) [reply gatsby-config.js:14]: I ❤️ this joke lol

**Pull request merged**